### PR TITLE
Addisie md004

### PR DESCRIPTION
**Pull request by Addisie**
As the Leporello is not measured I couldn't provide a measurement of it.

### CODE REVIEW

File: PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml
Comment (karljonaskarlsson) [inline PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:66]: I always try to mark up Geez words like this: `<foreign xml:lang="gez"> ተማህፀንኩ </foreign>`

File: PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml
Comment (karljonaskarlsson) [inline PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:74]: I believe there might be a better way to markup Ṭalsam, so that they also become searchable (regardless of spelling etc). Do you remember, @CarstenHoffmannMarburg ?

**Review by karljonaskarlsson — COMMENTED** [review]
Good job! :)

File: PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml
Comment (CarstenHoffmannMarburg) [reply PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:74]: I have frequently marked such drawings in the following way:
```
<decoNote xml:id="d1" type="miniature">
                                <desc>Coloured <term key="Magic">magic picture</term>.</desc>
                            </decoNote>
```
I understood, that it is a miniature, when it has more colours that just red and black. With black or red or black and red, it is a drawing.

You can use the word Ṭalsam, as well. I propose to mark it up with the keyword "Magic" and to mark the word as foreign. I do not know, whether this markup is obligatory, but certainly an improvement as it makes the decorations searchable:

```
<decoNote type="drawing" xml:id="d1">
                               <locus target="#1r"/>
                               <desc><desc><term key="Magic"><foreign xml:lang="gez">Ṭalsam<foreign></term>.</desc>
</decoNote>
```

**Comment by eu-genia** [comment]
NB
      > @eu-genia, thank you so much for letting me know. I won't do it next time, but I don't find any other.

no problem, just make sure that when you do the pull request for `MD004 `it is in the `new `folder and not with other MD records

_Originally posted by @eu-genia in https://github.com/BetaMasaheft/Works/issues/1195#issuecomment-2373834775_

**Comment by eu-genia** [comment]
As we had discussed, also in person, all records created through the application are stored on `betamasaheft.eu` in the `new `folder. That means, if you create a stub through `betamasaheft.eu` you MUST download to the `new `folder in your github repository and merge from there. Only later you can move or rename records.

In case of manuscripts it is recommended to create them locally and **not** through the application, but if you had used the application already, there is no other choice but working in the `new `folder, not the folder named after the institution.

Since this pull request has not yet been merged please, on your computer, move the file from the folder called `manuscripts/PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/`
to the folder called
`manuscripts/new/`
(if you do not have a folder called new then you have to create it) and commit to this branch, thank you

File: PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml
Comment (eu-genia) [inline PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:42]: Is the whole text unclear or only some of the words/letters? It is rare that such a long passage is unreadable.

File: PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml
Comment (nafisa-valieva) [inline PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:9]: Very nice record, Addissie! May be you can give the title both in Geez and English? This is a new fashion Denis suggested :-)

**Review by nafisa-valieva — APPROVED** [review]

File: PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml
Comment (eu-genia) [reply PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:9]: I do not know about the fashion Denis suggested but we normally only provide full title in Fidal and in translation for works, not for manuscripts.

Comment (nafisa-valieva) [reply PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:9]: > I do not know about the fashion Denis suggested but we normally only provide full title in Fidal and in translation for works, not for manuscripts.

Ok! then we follow this! @Addisie, you can ignore my suggestion as clearified by Eugenia.

File: PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml
Comment (Addisie) [reply PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:42]: > Is the whole text unclear or only some of the words/letters? It is rare that such a long passage is unreadable.

The last panel is really hard for me to read, at least
DSC01281
.

Comment (CarstenHoffmannMarburg) [reply PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml:42]: I suggest marking only those letters with `<unclear>` that are not clearly legible and therefore uncertain.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by eu-genia — CHANGES_REQUESTED** [review]
Aa long as the record is in the wrong folder we cannot merge it as it would create a duplicate in the app

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
once this is merged, you can move it to the correct place and launch a new PR

**Pull request merged**
